content/cross-platform/index.en-us.md

```diff
@@ -4,6 +4,7 @@ title: "Cross-platform"
 weight: 10
 toc: false
 draft: false
+url: "en-us"
```

Comment:
Does capitalization matter? As when I go to https://gitea.io I get redirected to https://gitea.io/en-US/

Reply:
Seems like it doesn't matter, the links in the footer to localized version are all lowercase.

Reply:
Sounds good! Thanks for quick response.

Reply:
Well I thank you for that quick LGTM :)